meilisearch/client.py

```diff
@@ -672,3 +673,21 @@ def _valid_uuid(uuid: str) -> bool:
         )
         match = uuid4hex.match(uuid)
         return bool(match)
+
+    def create_snapshot(self) -> TaskInfo:
+        """Trigger the creation of a Meilisearch snapshot.
+
+        Returns
+        -------
+        Snapshot:
+            Information about the snapshot.
+            https://www.meilisearch.com/docs/reference/api/snapshots#create-a-snapshot
```

Comment:
This will return information about the task and not the snapshot.

```suggestion
        Returns
        -------
        task_info:
            TaskInfo instance containing information about a task to track the progress of an asynchronous process.
            https://www.meilisearch.com/docs/reference/api/tasks#get-one-task
```

Reply:
Ok changed it